articles/hosted-pages/login/index.md

```diff
@@ -111,11 +137,13 @@ webAuth.authorize({
 });
 ```
 
-## Configure Different Pages per Client
+## Configure Multiple Pages by Using Separate Tenants
 
-In some cases, you might have multiple apps and want to configure separate login pages for each. Since the hosted pages are configured in the [Dashboard](${manage_url}) at tenant level (every client app you have set up on a single tenant would use the same Hosted Login Page), you would have to create a new tenant for each client that requires a different hosted page.
+In some cases, you might have multiple apps and want to configure separate login pages for each. Since the hosted pages are configured in the [Dashboard](${manage_url}) at the tenant level (every client app you have set up on a single tenant would use the same Hosted Login Page), you would have to create a new tenant for each client that requires a different hosted page. 
 
-There are two ways to handle this. The first option is to create different tenants for each client or sets of clients that you need to have separate custom pages for. Alternatively, you can use the [Management API](/api/management/v2) to set up a custom page for the specific client rather than using the editor in the [Dashboard](${manage_url}).
+::: note
+Remember that creating a new tenant is only really a viable option for one organization that needs two separate sets of custom pages, and can also have two distinct sets of clients, users, etc. as these items cannot be shared between tenants.
+:::
```

Comment:
Jeff this is correct but I can't see anything related to this `the guidance should explain why you want a single login page (as part of a unified branding), instead of directly recommending to create a different Auth0 tenant`

https://auth0.slack.com/archives/C5YSB7YLQ/p1503507409000038?thread_ts=1503479436.000301&cid=C5YSB7YLQ

Can you go over this thread and update this doc based on that input?